Project.toml

```diff
@@ -40,7 +40,7 @@ Missings = "0.4.2, 1"
 OffsetArrays = "1.12.10"
 PooledArrays = "1.4.2"
 PrecompileTools = "1"
-PrettyTables = "2.1"
+PrettyTables = "3.0.5"
```

Comment:
For compatibility with the rest of the Julia ecosystem, please support both v2 and v3, at least while people are still updating.

This could be as simple as putting the changes in this PR behind an `if pkgversion(PrettyTables).major == 3` check.

Reply:
Done! Of course I did not make the tests work in v2 since it seems unnecessary (the CI will always use v3 and it will add a lot of confusion IMHO).